src/screens/journeysDisplay.tsx

```diff
@@ -43,6 +22,22 @@ const JourneysScreen: React.FC<JourneysScreenProps> = ({
   route,
 }) => {
   const details = route.params.journeysDetails;
+  const mapJourneys = (detail: OutboundJourneysEntity, index: number) => {
```

Comment:
I like that this has been pulled out into a method! I think it could be clearer in its name about what it does - what it is it mapping a journey to?